eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -1323,4 +1323,70 @@ public function getSearchableFieldMapData()
 
         return $statement->fetchAll(\PDO::FETCH_ASSOC);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function updateContentTypeCopyIdentifier($typeId, $baseIdentifier)
```

Comment:
Taking back what I've said above, if we use `$typeId` of a just created copy, then I guess we're fine. Sorry for my confusion ;)